<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>69</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="colorComboBox">
     <item>
      <property name="text">
       <string>red</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>orange</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>yellow</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>green</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>teal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>blue</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>purple</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="deleteButton">
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QTimeEdit" name="timeEdit">
     <property name="displayFormat">
      <string>h:mm:ss</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
